### DISCUSSION TITLE
Failed to install this package

### DISCUSSION THREAD

**Discussion by EEOEON** [Onemind-Services-LLC/react-icons-ng#39] [Q&A]
![image](https://github.com/Onemind-Services-LLC/react-icons-ng/assets/28865214/72733a4a-dda2-47e7-8ce0-2790b9a320e0)

Hi! I tried to download this package because it offers more icons and is easier to search than react-icons. However, I can't install it. When I search on yarn and npm, it doesn't show up. How can I install it?

**Comment by abhi1693** [comment]
It's hosted via GitHub npm registry. Please read the readme for installation instructions https://github.com/Onemind-Services-LLC/react-icons-ng?tab=readme-ov-file#-installation-for-modern-projects
